include/fmt/format.h

```diff
@@ -43,6 +41,8 @@
 #include <memory>
 #include <stdexcept>
 
+#include "core.h"
```

Comment:
Please revert this as well.

Reply:
This is following the project's style as per [CONTRIBUTING.md](https://github.com/fmtlib/fmt/blob/ebec00138bd10e949ff34955bcb096387cd32acc/CONTRIBUTING.md), which states to follow the Google Style with a few listed exceptions.

The Google Style [order of includes](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) has project headers at the bottom.

I did not manually do this; this was done by `clang-format` which used the [.clag-format](https://github.com/fmtlib/fmt/blob/071794ec654a7c503b6214de4032c8ec8a07440b/.clang-format) file for this project; where [line 4](https://github.com/fmtlib/fmt/blob/071794ec654a7c503b6214de4032c8ec8a07440b/.clang-format#L4) indicates the Google style which made that change.

If it is intended, for the includes to have a different order; then it should be noted in `CONTRIBUTING.md`, and the relevant options set in the `.clang-format` file (unfortunately I cannot link directly to the options, in the [docs](https://clang.llvm.org/docs/ClangFormatStyleOptions.html), but I believe a mix of `IncludeBlocks` and `IncludeCategories` will allow the customisation).

The final files can then be re-run through `clang-format`, once the project has decided and implemented the format.

Reply:
Looks like some `clang-format` issue. Which version do you use?

Reply:
I do not believe this is an issue with clang format (I am using `clang-format 9.0.0`)
The Google Style is to have the project includes after all others; and the clang-format in the project root indicates that it is based off of the Google Style, and does not override those options.

The only way I can see there being a conflict, is if you have an untracked `.clang-format` in your copy of the repo at `include/.clang-format` and/or `include/fmt/.clang-format`, with different options, if you are using the automatic detection; or you a pointing it manually to the wrong style file (which I doubt is the case)

Reply:
Looks like they added include reordering recently. Anyway, I think it's fine since, as you correctly pointed out, this does follow the style guide.